StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1601UnitTests.cs

```diff
@@ -66,7 +66,7 @@ public async Task TestPartialTypeWithoutDocumentationAsync(string typeKeyword)
   ""settings"": {{
     ""documentationRules"": {{
       ""documentExposedElements"": false,
-      ""{interfaceSettingName}"": false
+      ""{interfaceSettingName}"": ""false""
```

Comment:
📝 This shouldn't change. It should continue to allow `false` and `true` (unquoted) for compatibility purposes.

```suggestion
      ""{interfaceSettingName}"": false
```